components/scream/src/physics/p3/tests/p3_nc_conservation_tests.cpp

```diff
@@ -52,7 +56,7 @@ struct UnitWrap::UnitTest<D>::TestNcConservation {
         nc_selfcollect_tend[s] = cxx_device(vs).nc_selfcollect_tend;
       }
 
-      Functions::nc_conservation(nc, nc_selfcollect_tend, cxx_device(0).dt, nc_collect_tend, nc2ni_immers_freeze_tend, nc_accret_tend, nc2nr_autoconv_tend);
+      Functions::nc_conservation(nc, nc_selfcollect_tend, cxx_device(offset).dt, nc_collect_tend, nc2ni_immers_freeze_tend, nc_accret_tend, nc2nr_autoconv_tend);
```

Comment:
@bartgol , this is related to the packed vs. non-packed inputs:

> When testing calls that take packed data, if those calls also take scalars, those scalars should be held constant.

Since we are holding the scalars constant, there's no reason to hardcode to the 0th dt, they're all the same. The behavior is the same either way.